pipelines/skylab/slideseq/SlideSeq.changelog.md

```diff
@@ -1,3 +1,8 @@
+# 3.4.4
+2024-11-04 (Date of Last Commit)
+
+* Changed a flag in JoinMultiomeBarcodes task in H5adUtils.wdl to se CSI instead of TBI indexing in tabix command to support chromosomes larger than 512 Mbp. This task should not affect the Slide-seq pipeline.
```

Comment:
```suggestion
* Updated the tabix flag in JoinMultiomeBarcodes task in H5adUtils.wdl to use CSI instead of TBI indexing, which supports chromosomes larger than 512 Mbp; this task should not affect the Slide-seq pipeline
```